From f4455e34c0f570b6a1f731034ca3c6f790dff763 Mon Sep 17 00:00:00 2001
From: Ryan Volz <rvolz@mit.edu>
Date: Wed, 20 May 2020 13:13:28 -0400
Subject: [PATCH 2/2] Don't build a win32 app instead of a console app.

---
 src/CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 8269184..2b99c9f 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -71,11 +71,5 @@ if(NOT Gnuradio_VERSION VERSION_LESS "3.8")
     )
 endif()
 
-#build a win32 app, not a console app
-if (WIN32)
-    set(CMAKE_EXE_LINKER_FLAGS "/entry:mainCRTStartup ${CMAKE_EXE_LINKER_FLAGS}")
-    set_target_properties(${PROJECT_NAME} PROPERTIES WIN32_EXECUTABLE ON)
-endif (WIN32)
-
 set(INSTALL_DEFAULT_BINDIR "bin" CACHE STRING "Appended to CMAKE_INSTALL_PREFIX")
 install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION ${INSTALL_DEFAULT_BINDIR})
-- 
2.25.1

