From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Mon, 12 Feb 2024 12:45:17 -0500
Subject: [PATCH] Don't build a win32 app instead of a console app.

---
 src/CMakeLists.txt | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index f0dcdb1..0209f66 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -109,15 +109,5 @@ else()
     )
 endif()
 
-#build a win32 app, not a console app
-if (WIN32)
-    if (MSVC)
-        set(CMAKE_EXE_LINKER_FLAGS "/entry:mainCRTStartup ${CMAKE_EXE_LINKER_FLAGS}")
-    else()
-        set(CMAKE_EXE_LINKER_FLAGS "-Wl,-e,mainCRTStartup ${CMAKE_EXE_LINKER_FLAGS}")
-    endif(MSVC)
-    set_target_properties(${PROJECT_NAME} PROPERTIES WIN32_EXECUTABLE ON)
-endif (WIN32)
-
 set(INSTALL_DEFAULT_BINDIR "bin" CACHE STRING "Appended to CMAKE_INSTALL_PREFIX")
 install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION ${INSTALL_DEFAULT_BINDIR})
-- 
2.42.0

