From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <rvolz@mit.edu>
Date: Wed, 20 May 2020 13:13:28 -0400
Subject: [PATCH] Don't build a win32 app instead of a console app.

---
 src/CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 1bf513a..3632427 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -84,11 +84,5 @@ else()
     )
 endif()
 
-#build a win32 app, not a console app
-if (WIN32)
-    set(CMAKE_EXE_LINKER_FLAGS "/entry:mainCRTStartup ${CMAKE_EXE_LINKER_FLAGS}")
-    set_target_properties(${PROJECT_NAME} PROPERTIES WIN32_EXECUTABLE ON)
-endif (WIN32)
-
 set(INSTALL_DEFAULT_BINDIR "bin" CACHE STRING "Appended to CMAKE_INSTALL_PREFIX")
 install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION ${INSTALL_DEFAULT_BINDIR})
-- 
2.34.1

